e2e/spec/login/login.e2e-spec.ts

```diff
@@ -9,7 +9,7 @@ describe("CATcher's Login Page", () => {
   });
 
   it('displays "CATcher" in header bar', async () => {
-    expect(await page.getTitle()).toEqual('CATcher');
+    expect(await page.getTitle()).toContain('CATcher');
```

Comment:
Why not verify the version number in the header too?

Reply:
I will add that check in too! ><

Reply:
We could perform the check as ```toEqual(`CATcher v${version}`)``` where `version` is obtained from `package.json`